src/script/page/collection/fullSearch/FullSearchItem.tsx

```diff
@@ -36,8 +37,23 @@ const FullSearchItem: React.FC<FullSearchItemProps> = ({message, onClick, format
   const {user, timestamp} = useKoSubscribableChildren(message, ['user', 'timestamp']);
   const {name} = useKoSubscribableChildren(user, ['name']);
   const {parts, matches} = formatText((message.getFirstAsset() as Text).text);
+
+  const handleSearchItemKeydown = (event: React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
this same (more or less) function appears a few times, can we move it to the keyboard utils?

Reply:
maybe just receives the function to be called as the second param